src/pages/contract/ContractPage.tsx

```diff
@@ -134,7 +134,7 @@ export function ContractPage(): JSX.Element {
           </div>
         </div>
         <div className="row white-row">
-          <div className="col">
+          <div className="col-12 col-sm-12 col-md-6 col-xl-6">
```

Comment:
Looking at https://getbootstrap.com/docs/4.1/layout/grid/#grid-options, it seems like we only need to set two classes: `col-12` for the smallest case and `col-md-6` for ≥768px.

Reply:
Bootstrap 4.5 version of the docs is here: https://getbootstrap.com/docs/4.5/layout/grid/#grid-options

See also #32

Reply:
The " <576px " for `.col-12` is misleading, since it is used for all screen sizes and then overridden. It should be documented as `≥ 0px`.

<img width="200" alt="Bildschirmfoto 2020-12-16 um 13 35 20" src="https://user-images.githubusercontent.com/2603011/102349424-98c1f800-3fa3-11eb-8f9a-829000f0319f.png">